include/HttpRequest.hpp

```diff
@@ -22,7 +22,8 @@ class HttpRequest : public HttpMessage
     // Getters & Setters
     const std::string& target() const;
     std::string& target();
-
+    void    set_target( const std::string& target );
```

Comment:
This function is not necessary, the target can be set by calling `target() = "new/target"`.